db/migrate/20160505005253_add_send_count_to_invitations.rb

```diff
@@ -0,0 +1,5 @@
+class AddSendCountToInvitations < ActiveRecord::Migration
+  def change
+    add_column :invitations, :send_count, :integer, null: true, default: 0
```

Comment:
Yeah I was kinda wondering that too.

On Fri, May 6, 2016 at 7:16 AM, Robert Guthrie <notifications@github.com>
wrote:

> In db/migrate/20160505005253_add_send_count_to_invitations.rb
> <https://github.com/loomio/loomio/pull/3328#discussion_r62239051>:
>
> > @@ -0,0 +1,5 @@
> > +class AddSendCountToInvitations < ActiveRecord::Migration
> > +  def change
> > +    add_column :invitations, :send_count, :integer, null: true, default: 0
>
> whatddya reckon about send_count being null: false, default 0? We don't
> really need that null value for legacy invites, they can be 0's
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3328/files/7f84c8234ec88a15a768fb37ce70fc44e073fa56#r62239051>
>